docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -98,6 +99,65 @@ for instance the MediaBundle defines the method as::
         }
     }
 
+.. note::
+    In your own admin, media is just a field and not the ``$object``. Therefore,
+    the code above must be updated this way:
+
+    .. code-block:: php
+
+        <?php
+
+            // ...
+
+            public function getObjectMetadata($object)
```

Comment:
the indention is wrong, it needs to be on the same position like the `<?php`-tag. Same for the other php block below